conanfile.py

```diff
@@ -40,7 +40,7 @@ def build_requirements(self):
         self.build_requires("gtest/1.14.0")
 
     def requirements(self):
-        self.requires("homestore/[~=6.0,      include_prerelease=True]@oss/master")
+        self.requires("homestore/[~=6.2,      include_prerelease=True]@oss/master")
```

Comment:
just use  "homestore/[~=6, include_prerelease=True]@oss/master" , then it will use the latest version of homestore 6.x